packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1619,15 +1619,45 @@ class CrudStoreImpl
         ...this.state.bulkDelete,
         status: 'in-progress',
       },
-    }); // TODO: COMPASS-7328
+    });
+
+    openToast('bulk-delete-toast', {
+      title: '',
+      variant: 'progress',
+      dismissible: true,
+      timeout: null,
+      description: `${
+        this.state.bulkDelete.affected || 0
+      } documents are being deleted.`,
+    });
   }
 
   bulkDeleteFailed(ex: Error) {
-    return ex; // TODO: COMPASS-7328
+    openToast('bulk-delete-toast', {
+      title: '',
+      variant: 'warning',
+      dismissible: true,
+      timeout: 6_000,
+      description: `${
+        this.state.bulkDelete.affected || 0
+      } documents could not be deleted.`,
+    });
+
+    log.error(mongoLogId(1_001_000_268), 'Bulk Delete Documents', ex.message, {
```

Comment:
```suggestion
    log.error(mongoLogId(1_001_000_268), 'Bulk Delete Documents', `Delete opeartion failed: ${ex.message}`, {
```

That’s a bit more in line with our usual error messages in logs, the raw error message might not be all that informative depending on what it ends up being :)